azext_iot/common/_azure.py

```diff
@@ -12,64 +12,71 @@
 
 
 def _get_aad_token(cmd, resource=None):
```

Comment:
This component is in azext_iot/common/auth.py now. Would be good to remove here and reference that one.